lib/sync/installation.js

```diff
@@ -102,14 +102,21 @@ module.exports.processInstallation = (app, queues) => {
       return processor(github, repository, cursor, perPage)
     }
 
+    const handleGitHubError = (err) => {
+      const ignoredErrorTypes = ['MAX_NODE_LIMIT_EXCEEDED']
+      const notIgnoredError = err.errors.filter(error => !ignoredErrorTypes.includes(error.type)).length
```

Comment:
I'm not gonna block, but I wanted to point out one thing to keep in mind 😄 

You're using a `!` inside the filter and a `not` in the var name... all the nots make it more difficult to parse. I get what you're doing here, and it's limited scope, but be careful in a larger setting ✨

Reply:
👋 Thanks for bringing this up @jules2689 and @jnraine! Totally understand the confusion here and I can see how the naming is not ideal. 

Since there can be multiple errors on the response (at least there are on the Axios return object) I filter the errors to check if we have **any** error type that we shouldn't be ignoring. I figured we don't want to swallow the error if there is a different error with a non-ignored error type. If I invert the variable then we'd swallow the error regardless of any of the other error types.

I'm building off this PR and this section of the codebase to catch timeout errors in another upcoming PR so I'll revisit it again to see if there's a better way to name this 😆